content/en/monitors/service_level_objectives/guide/slo_graph_query.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Scope metric-based SLO queries
+kind: guide
+disable_toc: false
+further_reading:
+- link: "/dashboards/widgets/slo/"
+  tag: "Documentation"
+  text: "SLO Summary Widget"
+---
+
+<div class="alert alert-info">This feature is only available for <strong>metric-based</strong> SLO queries.</div>
+
+## Overview
+
+The [SLO Summary widget][1] supports advanced metric query filtering, including the use of template variables to dynamically scope results displayed. 
+
+## Walk through of an SLO query
```

Comment:
One minor suggestion that I have is to add a sentence, either here or in the overview that describes the process outlined in this article (e.g., "This guide describes how to..."). You could also modify the first sentence under the Metric-based SLO query heading and have it say "First, create a metric-based SLO." 

My reason for this suggestion is that it wasn't entirely clear to me what the steps were when I first read the article, i.e., you first create the metric-based SLO query, and then you add it to the SLO Summary widget.

Reply:
Makes sense!